convert2rhel/actions/post_conversion/hostmetering.py

```diff
@@ -0,0 +1,235 @@
+            logger.critical_no_exit("host-metering service is not running.")
+            self.set_result(
+                level="ERROR",
+                id="HOST_METERING_NOT_RUNNING",
+                title="Host metering service is not running.",
+                description="host-metering.service is not running.",
+                remediations="You can try to start the service manually"
+                " by running following command:\n"
+                "   systemctl start host-metering.service",
+            )
+
+        return service_running
+
+    def _check_env_var_correct(self):
+        """Check if the env var is set and if it's right name"""
+        if self.env_var is None:
+            logger.debug("CONVERT2RHEL_CONFIGURE_HOST_METERING was not set. Skipping it.")
+            self.add_message(
+                level="INFO",
+                id="SKIPPED_CONFIGURE_HOST_METERING",
+                title="Skipping host metering configuration.",
+                description="CONVERT2RHEL_CONFIGURE_HOST_METERING was not set.",
+            )
+            return False
+
+        if self.env_var not in ("force", "auto"):
+            logger.debug("Value for environment variable not recognized: %s" % self.env_var)
+            self.add_message(
+                level="WARNING",
+                id="UNRECOGNIZED_OPTION_CONFIGURE_HOST_METERING",
+                title="Unrecognized option in CONVERT2RHEL_CONFIGURE_HOST_METERING environment variable.",
+                description="Environment variable {env_var} not recognized.".format(env_var=self.env_var),
+                remediations="Set the option to `auto` value if you want to configure host metering.",
+            )
+            return False
+
+        return True
+
+    def _set_should_configure_metering(self):
+        if self.env_var == "force":
+            self.should_configure_metering = True
+            logger.warning(
+                "The `force' option has been used for the CONVERT2RHEL_CONFIGURE_HOST_METERING environment variable."
+                " Please note that this option is mainly used for testing and will configure host-metering unconditionally. "
+                " For generic usage please use the 'auto' option."
+            )
+            self.add_message(
+                level="WARNING",
+                id="FORCED_CONFIGURE_HOST_METERING",
+                title="The `force' option has been used for the CONVERT2RHEL_CONFIGURE_HOST_METERING environment variable.",
+                description="Please note that this option is mainly used for testing and"
+                " will configure host-metering unconditionally."
+                " For generic usage please use the 'auto' option.",
+            )
+        elif self.env_var == "auto":
+            self.should_configure_metering = True
+            logger.debug("Automatic detection of host hyperscaler and configuration.")
+        else:
+            self.should_configure_metering = False
```

Comment:
We never reach this as we always verify `env_var`. If it fails verification we return right away. Do we even need to set `self.should_configure_metering` as it will always be `True` if we execute?
```suggestion
```

Reply:
Good point, it was already present in the code and I didn't want to touch the logic much. But it seems it can be removed or reworked